jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/AccessAutoConfiguration.java

```diff
@@ -38,5 +40,10 @@ public AccessAutoConfiguration(AccessProperties accessProperties) throws Invalid
 		logger.info("Input plugin: {}", accessProperties.getInput().getPlugin());
 		logger.info("Output plugin: {}", accessProperties.getOutput().getPlugin());
 	}
+
+	@Bean(name="inputConfig")
```

Comment:
explicit is better, right ;-)

Reply:
yup I prefer it! 😄 magical things happening due to frameworks are always hard to understand haha